tools/congestion-model/src/model/mod.rs

```diff
@@ -124,6 +124,16 @@ impl Model {
             .collect()
     }
 
+    pub fn trim_transaction_pools(&mut self, max_len: usize) {
+        for &shard_id in &self.shard_ids {
+            if self.queues.incoming_transactions(shard_id).len() > max_len {
+                self.queues
+                    .incoming_transactions_mut(shard_id)
+                    .resize_with(max_len, || unreachable!());
+            }
```

Comment:
Does that trim the newest or oldest transactions? I think we should trim the old ones and keep the fresh ones.

Reply:
I see, that makes sense.

I have been using it with 0 or 10, in those cases it's all transactions from this round.
But I can see other cases making sense as well. Changing it right now.

Reply:
done